client/my-sites/upgrades/cart/cart-plan-discount-ad.jsx

```diff
@@ -50,10 +51,35 @@ const CartPlanDiscountAd = React.createClass( {
 			return null;
 		}
 
+		if ( plan.isDomainUpgrade ) {
+			return (
+				<CartAd>
+					<p>{ translate(
+						"You're getting a %(discount)s discount off the regular price of the plan (%(originalPrice)s)" +
+						", because you already paid for the domain.",
+						{
+							args: {
+								discount: plan.formattedDiscount,
+								originalPrice: plan.formattedOriginalPrice
+							}
+						}
+					) }</p>
+					<p>{ translate(
```

Comment:
The screenshot you posted above looks good! The padding top/bottom is same. Check my screenshots.